### ISSUE TITLE
Use with Shoutcast

### ISSUE THREAD

**Issue by gauuni** [douban/DOUAudioStreamer#110]
Hi,
I am using your Project to streaming from Shoutcast. But I don't know how?
Can you explain to me?

Thanks.

**Comment by iDevelopper** [comment]
@gauuni , did you find how?

**Comment by gauuni** [comment]
Yes, shoutcast give us a playlist file. You should read it to get direct link and stream.
Good luck

**Comment by iDevelopper** [comment]
You mean a .PLS file? In the playlist file, there is a stream url, not a link to a file. How can I do please?
